ocaml/utils/warnings.ml

```diff
@@ -581,6 +583,10 @@ let descriptions = [
     names = ["unboxing-impossible"];
     description = "The parameter or return value corresponding @unboxed attribute cannot be unboxed.";
     since = since 4 14 };
+  { number = 211;
+    names = ["mod-by-top"];
+    description = "Modifying by the top element of an axis is a no-op.";
```

Comment:
I think "kind" should be mentioned here somewhere

Reply:
done